pontoon/translations/views.py

```diff
@@ -450,17 +499,41 @@ def reject_translation(request):
         plural_form=translation.plural_form,
     )
 
-    # Reward Review Master Badge stats
-    if request.user.badges_review_count in settings.BADGES_TRANSLATION_THRESHOLDS:
-        # TODO: Send a notification to the user
-        pass
-
-    return JsonResponse(
-        {
-            "translation": active_translation.serialize(),
-            "stats": TranslatedResource.objects.stats(project, paths, locale),
+    # Send Review Master Badge notification information
+    response_data = {
+        "translation": active_translation.serialize(),
+        "stats": TranslatedResource.objects.stats(project, paths, locale),
+    }
+
+    review_count = request.user.badges_review_count
+    if review_count in settings.BADGES_TRANSLATION_THRESHOLDS:
+        response_data["badge_update"] = {
+            "name": "Review Master Badge",
+            "level": settings.BADGES_TRANSLATION_THRESHOLDS.index(review_count) + 1,
         }
-    )
+        desc = """
+        You have gained a new badge level!
+        <br>
+        Review Master Badge: Level {level}
+        <br>
+        You can view this badge on your <a href={profile_href}> profile page </a>.
+        """.format(
+            level=settings.BADGES_TRANSLATION_THRESHOLDS.index(review_count) + 1,
+            profile_href=reverse(
+                "pontoon.contributors.contributor.username",
+                kwargs={
+                    "username": request.user.username,
+                },
+            ),
+        )
+        notify.send(
+            sender=request.user,
+            recipient=request.user,
+            verb="",  # Triggers render of description only
```

Comment:
Isn't this the same notification we already send in [`pontoon/base/forms.py`](https://github.com/mozilla/pontoon/pull/3442/files#diff-3f6e1499881b2916dc46a23905b95100528c9221f79f5f4c1827f486bec596ad)?

Reply:
Yes, should there be a generic helper function for sending out badge notifications?